packages/kbn-plugin-generator/src/ask_questions.ts

```diff
@@ -32,6 +32,11 @@ export const INTERNAL_PLUGIN_LOCATIONS: Array<{ name: string; value: string }> =
     name: 'Kibana OSS',
     value: Path.resolve(REPO_ROOT, 'src/plugins'),
   },
+  {
```

Comment:
This one splits in two subfolders, `'internal' | 'shared'` I wonder if I should make both explicit here.
(IDEM for _X-Pack Platform_)